apps/desktop-wallet/src/components/Popup.tsx

```diff
@@ -131,7 +131,7 @@ const Content = styled(motion.div)<Pick<PopupProps, 'minWidth'>>`
   flex-direction: column;
 
   min-width: ${({ minWidth }) => minWidth}px;
-  max-height: 510px;
+  max-height: 660px;
```

Comment:
I was looking for the place in the code where we define the min width and height of the app, but I couldn't find it anywhere. Do you know where it is? I am asking because I know that we don't allow resizing the window below a certain threshold. Just curious.

Reply:
In `electron.js` if I remember correctly. 700px I believe.

Reply:
great memory 🧠 https://github.com/alephium/alephium-frontend/blob/master/apps/desktop-wallet/public/electron.js#L163